app/src/main/java/com/example/pehchaan/fragment/HomeFragment.java

```diff
@@ -155,4 +149,9 @@ public void onClick(View v) {
         return v;
 
     }
+
+    private void addContact() {
```

Comment:
Change this method name to 'openAddContactScreen'

Reply:
@soumenkp2 
resolved